freeipa/src/main/resources/freeipa-salt/salt/freeipa/common-install.sls

```diff
@@ -65,3 +65,19 @@ restart_httpd:
     - watch:
       - file: /etc/httpd/conf.d/ipa-rewrite.conf
 
+install-healthagent:
```

Comment:
shouldn't this state have any onlyif/unless option, so we won't run it if not necessary?

Reply:
There was an unless option before, but I think there was a timing problem between the unless call and the actual script because it should not have run based on the unless, but things were failing because it was.  I moved the unless check into the script itself so the script may be called but the actual action isn't performed unless it's necessary.

Reply:
this will cause saltstack will always detect this as a change which is not ideal. we can still place a file somewhere from the script and check that from unless

Reply:
updated to a better check.

Reply:
I think you should place the unless script in the require part too

Reply:
added